lib/driver.js

```diff
@@ -247,7 +247,7 @@ class AndroidDriver extends BaseDriver {
         if (await this.adb.getApiLevel() >= 28) { // API level 28 is Android P
           // Don't forget to reset the relaxing in delete session
           log.warn('Relaxing hidden api policy to manage animation scale');
-          await this.adb.setHiddenApiPolicy('1');
+          await this.adb.setHiddenApiPolicy('1', this.opts.ignoreHiddenApiPolicyError || false);
```

Comment:
`|| false` is not needed
you could simply do `!!this.opts.ignoreHiddenApiPolicyError` to make sure it is a boolean value

Reply:
same comment is relevant for the two other linked PRs

Reply:
Yes. Will fix them after this new version push